### PR TITLE
feat: Made the navbar stick to the top.

### PR DESCRIPTION
**Pull request by nagalakshmi08**
**What kind of change does this PR introduce?**
I had made the navbar stick to the top so that it can be accessible from anywhere of the page.

**Issue Number:**
Fixed issue #11

### CODE REVIEW

**Comment by Doraemon012** [comment]
@Joyosmit
Please provide a screen recording of the final output.

Also remove the changes in `firebaseConf.ts`. Remember that the `firebase configuration` is confidential, so never push it to any public repository.

File: src/components/NavBar/NavBar.tsx [hunk 1]
```diff
@@ -25,7 +25,8 @@ const NavBar = () => {
   const userEmailId = localStorage.getItem("userEmailId");
 
   return (
-    <Navbar bg="dark" variant="dark" expand="lg" expanded={expanded} style={{ userSelect: 'none' }}>
+    <Navbar bg="dark" variant="dark" expand="lg" expanded={expanded} style={{ userSelect: 'none' }}
```
Comment (Doraemon012) [inline src/components/NavBar/NavBar.tsx:28]: Add React Bootstrap's Position Utility to make the navbar sticky.
 
Use this as reference: [React Bootstrap Navbar](https://react-bootstrap.netlify.app/docs/components/navbar/#sticky-top)

**Review by Doraemon012 — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by Doraemon012** [comment]
image

@nagalakshmi08 
 Your implementation is correct, but this can be done more easily using react bootstrap.
 You just need to add React Bootstrap's Position Utility to make the navbar sticky.
 
 Use this as reference: [React Bootstrap Navbar](https://react-bootstrap.netlify.app/docs/components/navbar/#sticky-top)

**Comment by nagalakshmi08** [comment]
> @nagalakshmi08 Please provide a screen recording of the final output.
> 
> Also remove the changes in `firebaseConf.ts`. Remember that the `firebase configuration` is confidential, so never push it to any public repository.


https://github.com/Tanay-ErrorCode/lupo-skill/assets/121669832/2d903c05-dfce-4c0c-8e11-d615417d648c

This is the recording of the webpage. 
Screenshot 2024-05-13 212134
When I opened the events page, no events are being shown that's why I have used sample data to test the navbar.
Also I will remove the firebaseConf.ts and update the repo, I have added then because I got error while running the code without firebase details.

**Comment by nagalakshmi08** [comment]
> image
> 
> @nagalakshmi08 Your implementation is correct, but this can be done more easily using react bootstrap. You just need to add React Bootstrap's Position Utility to make the navbar sticky.
> 
> Use this as reference: [React Bootstrap Navbar](https://react-bootstrap.netlify.app/docs/components/navbar/#sticky-top)

Sure I will implement using bootstrap and update the code.

**Comment by Doraemon012** [comment]
> When I opened the events page, no events are being shown that's why I have used sample data to test the navbar. Also I will remove the firebaseConf.ts and update the repo, I have added then because I got error while running the code without firebase details.


1. The reason you didn't see any events on the events page is because there were no events added to the Firebase database. When you run the website locally, any events you create through the site are stored in your own Firebase database. Since the database starts empty and you hadn't added any events yet, the page appeared blank.

2. You're right about getting an error without the Firebase details. When you run the code without providing your Firebase configuration, the local website won't be able to connect to Firebase, hence the error.

3. Just a reminder, your Firebase details are only necessary for running the project locally on your computer. They shouldn't be shared publicly or pushed to any public platform.

**Review by Doraemon012 — APPROVED** [review]

**Comment by Doraemon012** [comment]
@nagalakshmi08 Great work! Looks good to me.

**Pull request merged**